Code/Tools/Libs/GuiFoundation/PropertyGrid/Implementation/AddSubElementButton.cpp

```diff
@@ -141,32 +139,6 @@ void ezQtAddSubElementButton::onMenuAboutToShow()
     }
     m_SupportedTypes.Insert(pProp->GetSpecificType());
 
-    ezVariant constraintValue;
```

Comment:
Why was this removed?

Reply:
This feature was not used anywhere so I removed the corresponding attribute and the code here.